### PR TITLE
Displaying Capabilities on Page US010

### PR DESCRIPTION
**Pull request by ChrisThompsonK**
Updated capabilities UI

Added backend for pulling capability information out based on ID. also added alias to SQL command to reference capability object

Adding description column and descriptions for capabilities

Changed the picture links in dummy data to file paths, and have added the functionality for pictures to be displayed

Pushing local changes of webpage

Updated roles section UI in the capability page

Updated design of compatibility page

Adding url params for capabilites

moved styling to css

### CODE REVIEW

**Review by Conormc98 — APPROVED** [review]

**Review by micalexis — APPROVED** [review]

File: ExpressAPI/db.js [hunk 1]
```diff
@@ -189,9 +189,9 @@ exports.clearUserToken = function(tokenToClear){
...
 
 
-exports.getCapabilities = function(callback) {
+exports.getCapabilities = function(ID, callback) {
     db.query(
-        "SELECT capability.ID, capability.name, capLead.name AS leadName, capLead.photo, capLead.message FROM capability INNER JOIN capLead ON capability.leadID=capLead.ID;",
+        "SELECT capability.ID AS ID, capability.name AS name, capability.description AS capDescription, capLead.name AS leadName, capLead.photo AS photo, capLead.message AS message FROM capability INNER JOIN capLead ON capability.leadID=capLead.ID WHERE capability.ID=?", [ID],
         function(err, rows) {
             if (err) {
                 logger.error("getCapabilities failed with error: " + err)
```
Comment (JamesMatchett) [inline ExpressAPI/db.js]: ;

File: ExpressAPI/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ app.get('/roles', function (req, res) {
 });
 
 app.get('/capabilities', function(req, res) {
-    db.getCapabilities(function(rows) {
+    db.getCapabilities(req.query.capabilityID, function(rows) {
         res.send(rows)
```
Comment (JamesMatchett) [inline ExpressAPI/index.js:28]: ;

File: kjourney/src/app/capability/capability.component.ts [hunk 3]
```diff
@@ -13,16 +14,19 @@ export class CapabilityComponent implements OnInit {
   private data: DataService;
   public roles: Role[];
   public capabilities: Capability[];
+  public capabilityID: number;
 
-  constructor(data: DataService) {
+  constructor(data: DataService, private route: ActivatedRoute) {
     this.data = data
```
Comment (JamesMatchett) [inline kjourney/src/app/capability/capability.component.ts]: ;

**Review by JamesMatchett — CHANGES_REQUESTED** [review]

**Review by JamesMatchett — APPROVED** [review]
נראה בן טוב, זה יחסי ציבור נאים אדירים

**Pull request merged**